src/wp-admin/options-general.php

```diff
@@ -179,7 +179,17 @@
 <tr>
 <th scope="row"><label for="default_role"><?php _e( 'New User Default Role' ); ?></label></th>
 <td>
-<select name="default_role" id="default_role"><?php wp_dropdown_roles( get_option( 'default_role' ) ); ?></select>
+	<?php
+	/**
+	 * Filters the roles included in the default_role option.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param array $roles_to_include Array of roles to include in the default_role dropdown. Defaults to subscriber, contributor and author.
+	 */
+	$roles_to_include = (array) apply_filters( 'default_user_dropdown_included_roles', array( 'subscriber', 'contributor', 'author' ) );
```

Comment:
Still the wrong filter name FWIW

Reply:
What should I name the filter?

Reply:
My suggested changes included the updated name.

It should be `default_role_dropdown_...`, not `default_user_dropdown_....`

Reply:
Thanks! Sorry, I overlooked that